app/src/main/kotlin/com/wire/android/core/extension/Fragment.kt

```diff
@@ -1,7 +1,14 @@
 package com.wire.android.core.extension
 
+import androidx.appcompat.app.AppCompatActivity
+import androidx.fragment.app.Fragment
 import androidx.fragment.app.FragmentManager
 import androidx.fragment.app.FragmentTransaction
 
 inline fun FragmentManager.inTransaction(func: FragmentTransaction.() -> FragmentTransaction) =
-    beginTransaction().func().commit()
\ No newline at end of file
+    beginTransaction().func().commit()
+
+//TODO: Fragment shouldn't be aware of activity's view ids
```

Comment:
Is it something that we can fix here instead of keeping the TODO?

Reply:
this requires a navigation structure, or a communication mechanism between the fragment & activity. so, no we can't fix it now :)